arduino/cores/packagemanager/install_uninstall.go

```diff
@@ -237,6 +238,9 @@ func (pme *Explorer) RunPostInstallScript(installDir *paths.Path) error {
 		if err != nil {
 			return err
 		}
+		cmd.RedirectStdoutTo(os.Stdout)
+		cmd.RedirectStderrTo(os.Stderr)
```

Comment:
I fear that is more complicated than this... this breaks the JSON output, and we must use the `feedback` package to avoid that.

Probably the most simple way is to pass two `io.Writers` to `RunPostInstallScript`:

```go
func (pme *Explorer) RunPostInstallScript(installDir *paths.Path, stdout, stderr io.Writer) error {
```

and redirect the output to those streams. Also, this is a breaking API change in golang to document in UPGRADING.md

Reply:
I see thanks for noticing. Let me check what writers would fit